smoke-tests/src/test/java/com/splunk/opentelemetry/GlassFishSmokeTest.java

```diff
@@ -34,8 +34,7 @@
   private static Stream<Arguments> supportedConfigurations() {
     return configurations("payara")
         .otelLinux("5.2020.6", PAYARA_SERVER_ATTRIBUTES, VMS_ALL, "8", "11")
-        .otelWindows("5.2020.6", PAYARA_SERVER_ATTRIBUTES, VMS_ALL, "8", "11")
-        .stream();
+        .otelWindows("5.2020.6", PAYARA_SERVER_ATTRIBUTES, VMS_ALL, "8", "11").stream();
```

Comment:
why did these smoke test changes make it into this PR?

Reply:
NO IDEA. Must have been a rebase thing, but I fail to understand how (did rebase on main only).